lib/engine/tile.rb

```diff
@@ -274,6 +274,19 @@ def token_blocked_by_reservation?(corporation)
       end
     end
 
+    def city_town_edges
+      # Returns a list of each edge a city/town goes to
```

Comment:
if the intention was to return a list, maybe you should do values at the end here intsead of calling it on 206 and 210 of the other file